### ISSUE TITLE
Subscriber Groups button are inaccessible/unclickable from sidebar in 2012 layout with PocketTube

### ISSUE THREAD

**Issue by Patricek0056** [lightbeam24/customtube#13]
### Problem description:
- The Subscriber Groups sidebar section (with PocketTube extension installed) is unclickable, which means, that it is inaccessible.
- Also the buttons in PocketTube are greyed out.
- The “Deck” button is though accessible, but greyed out too.

### Expected results:
- The Subscriber Groups in the sidebar should be accessible.
- The buttons in PocketTube should NOT be greyed out!

### Screenshots:
- **SCREENSHOT NOTE:** “Skupiny Odberov” means “Subscriber Groups”.
- **WARNING:** Inaccurate accessibility due to black arrows icons, black text & dark grey background in this screenshot!

![image](https://github.com/lightbeam24/customtube/assets/69351811/8d0212cf-f4bb-4294-83c8-c867c9c4ff18)

**Comment by lightbeam24** [comment]
I haven't used PocketTube, but based on your screenshot, it looks like PocketTube is creating its own sidebar elements that use separate styling from the native YouTube sidebar items. So basically this only happens because CustomTube wasn't tested with PocketTube. (The only other extensions/scripts CustomTube was tested with are uBlock Origin and Return YouTube Dislike.) 

I'll look into it though. In the meantime, could you tell me if it happens on other layouts? In this situation, it would be most helpful to test on the 2013, 2017, and 2019 layouts. If it happens on all three of those layouts, test it with "Layout Select" set to "Disabled".

**Comment by Patricek0056** [comment]
In the 2013, the PocketTube is not only inaccessible, but also large (Wait for me to testing the 2017, and 2019 please…)
![image](https://github.com/lightbeam24/customtube/assets/69351811/3d892a85-bf0c-457b-8adf-1f4552011fe7)

**Comment by Patricek0056** [comment]
2014 PocketTube is clickable, but the buttons are large, and Deck button is unclickable (Sorry, but you must wait, because I have some things to do with Firefox's CSS, so be patient):
![image](https://github.com/lightbeam24/customtube/assets/69351811/1aa98db8-1191-4051-be09-a8982f3eed6a)

**Comment by Patricek0056** [comment]
Well, 2015 is same as 2014. (Sorry for delay)

**Comment by Patricek0056** [comment]
2016 ditto as 2015, and 2014.

**Comment by lightbeam24** [comment]
Yeah 2014-2017 should all be the same, there isn't much of a need to test them all. They share most of their code.

**Comment by Patricek0056** [comment]
And for 2019, then there is wrong outline shade for Deck button, and the Sub. Collections are using wrong buttons shape, they should be square instead of rounded.

![image](https://github.com/lightbeam24/customtube/assets/69351811/e759fb57-a920-47ee-ab99-466f22c30921)

**Comment by lightbeam24** [comment]
Okay, so that adds weight to my theory that PocketTube uses it's own CSS to style its sidebar items. I'll look into fixing it.

**Comment by Patricek0056** [comment]
Please, I need your answer, is this blue tick mark feature of YouTube, or feature of PocketTube?
![image](https://github.com/lightbeam24/customtube/assets/69351811/8556a23e-df21-4c35-bbe7-0626e565f470)

If it is feature of YouTube, then it needs to be fixed, if it is feature of PocketTube, then I don't know.

**Comment by Patricek0056** [comment]
Fine, I will write it to description, and title.
